openapi-v2.yaml

```
@@ -1532,6 +1532,9 @@ definitions:
     description: "A union type which may contain a credential to access any one cloud provider."
     type: "object"
     properties:
+      uuid:
+        description: The unique UUID of the credential
```

Nit: The second "U" in "UUID" stands for unique.
```suggestion
        description: The UUID of the credential
```